QueryBuilder/Compilers/Compiler.cs

```diff
@@ -17,6 +17,11 @@ public partial class Compiler
         protected virtual string LastId { get; set; } = "";
         protected virtual string EscapeCharacter { get; set; } = "\\";
 
+
+        protected virtual string SingleInsertStartClause { get; set; } = "INSERT INTO";
```

Comment:
I think this is the case for Oracle only, I prefer removing it from the global compiler, unless it's really needed